presto-docs/src/main/sphinx/security/server.rst

```diff
@@ -0,0 +1,254 @@
+==========================================
+Presto Coordinator Kerberos Authentication
+==========================================
+
+The Presto server can be configured to enable Kerberos authentication over
+HTTPS for clients such as the :doc:`Presto CLI </security/cli>` or the various
+JDBC and ODBC drivers.
+
```

Comment:
Add comma after clients in second line, and after the reference (before "or"). 
Suggest deleting "the various."